styles.css

```diff
@@ -29,7 +29,120 @@ a {
 ul.nolist {
 	list-style-type: none;
 	margin: 0;
-	padding: 5px 15px;
+	padding: 0;
+}
+
+.button {
+	display: inline-block;
+	text-transform: uppercase;
+	font-size: 10px;
+}
+
+.button:after {
+	content: "";
+	display: inline-block;
+	margin: 0 5px;
+}
+
+.button:last-of-type:after {
+	margin: 0;
+	display: none;
+}
+
+/* Form Elements */
+
+.form-control {
+	border: 4px solid silver;
+	font-size: 20px;
+	outline: none;
+	padding: 20px 10px;
+}
+
+.form-control:focus {
+	border-color: gray;
+}
+
+select.form-control {
+	height: 48px;
+}
+
+.form-inline .form-group {
+	vertical-align: top;
+	margin-right: 10px;
+}
+
+label {
+	display: block;
+}
+
+.error-message {
+	color: red;
+	font-size:;
+}
+
+.form-radio label:first-of-type {
+	margin-bottom: 15px;
+}
+
+.form-radio input[type="radio"] {
+	padding: 0;
+	display: none;
```

Comment:
Custom radio buttons! This will "hide" the actual input. The input circle is then re-added via the property below: `form-radio fieldset label:before`